Flow/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 ossd-sp22
```

Comment:
Hmm, is this your name? 🤔

Reply:
Hi, we develop this theme in a team of four. I can change the copyright name to the GitHub username of the four of us.

Reply:
Hi, I've deleted the .idea file and changed the information in the license.